.github/workflows/settings-ci.yml

```
@@ -0,0 +1,67 @@
+name: Validate Settings
+
+on:
+  pull_request:
+    paths:
+      - "slack-archive.yaml"
+
+permissions:
+  pull-requests: write
+
+jobs:
+  check_files_changed:
+    runs-on: ubuntu-latest
+    outputs:
+      only_changed: ${{ steps.check.outputs.only_changed }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check if only slack-archive.yaml changed
+        id: check
+        run: |
+          files=$(git diff --name-only ${{ github.event.before }} ${{ github.sha }})
+          if [[ "$files" == "slack-archive.yaml" ]]; then
+            echo "only_changed=true" >> $GITHUB_ENV
```

envではなくoutputな気がしますelse blockも同じく

```suggestion
            echo "only_changed=true" >> $GITHUB_OUTPUT
```

---

本当ですね、やらかしてる
copilotの所為ということにしておきます